src/promptflow/tests/executor/e2etests/test_executor_validation.py

```diff
@@ -117,11 +117,7 @@ def test_bulk_run_input_type_invalid(self, flow_folder, batch_input, error_messa
         bulk_result = executor.exec_bulk(
             batch_input,
         )
-        if (
-            (sys.version_info.major == 3)
-            and (sys.version_info.minor >= 11)
-            and ((sys.platform == "linux") or (sys.platform == "darwin"))
-        ):
+        if (sys.version_info.major == 3) and (sys.version_info.minor >= 11):
```

Comment:
why removed the os conditions, as the line 121 still says linux and macos

Reply:
remove linux and macos restrictions, since win also has problems, we didn't find it in previous versions since tests are bypassed by misconfiguration.